Cabal/Distribution/Simple/Configure.hs

```diff
@@ -413,20 +445,28 @@ configure (pkg_descr0', pbi) cfg = do
                 allConstraints
                 (dependencySatisfiable
                     (fromFlagOrDefault False (configExactConfiguration cfg))
+                    (packageVersion pkg_descr0)
                     installedPackageSet
                     internalPackageSet
                     requiredDepsMap)
                 comp
                 compPlatform
                 pkg_descr0
 
+    debug verbosity $ "Finalized package description:\n"
+                  ++ showPackageDescription pkg_descr
+    -- NB: showPackageDescription does not display the AWFUL HACK GLOBAL
+    -- buildDepends, so we have to display it separately.  See #2066
+    debug verbosity $ "Finalized build-depends: "
+                  ++ intercalate ", " (map display (buildDepends pkg_descr))
```

Comment:
Ok. We should aim to really eliminate that `buildDepends` in cabal 2.0 refactoring.